Yearn-agents/PPSValues/src/agent.ts

```diff
@@ -0,0 +1,82 @@
+import BigNumber from "bignumber.js";
+import {
+  BlockEvent,
+  Finding,
+  HandleBlock,
+  FindingSeverity,
+  FindingType,
+  getJsonRpcUrl,
+} from "forta-agent";
+import Web3 from "web3";
+import { abi, getYearnVaults } from "./utils";
+
+const web3 = new Web3(getJsonRpcUrl());
+
+export const createFinding = (
+  ppo: string,
+  tracker: string,
+  reason: string,
+  id: number
+) => {
+  return Finding.fromObject({
+    name: "Yearn PPS Agent",
+    description: `Year PPS value: ${reason}`,
+    alertId: `Yearn-8-${id}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Unknown,
+    metadata: {
+      ppo,
+      tracker,
+    },
+  });
+};
+
+const provideHandleFunction = (web3: Web3): HandleBlock => {
+  let tracker = new BigNumber(1);
```

Comment:
why do we assume pps is 1 to start?

Reply:
Because we are calculating the increase in pps too. So if a value is 0, then any increment would result in infinity % change. I can add an exception condition, but didn't felt like it was required.